mobify/platform-scaffold
# Upgrade test:build-size

Web-team would like to verify the build files sizes do not get any bigger than they are now so we're upgrading our test:build-size!

With this test, we will fail if any file in the build folder exceeds the threshold of the size we set it to be.

We have a configurable file: file-size-config.json of which we will list a bunch of files and their gzipped filesize. We will be running this test through every CircleCI build.

Please note that we are comparing it against the predicted gzipped file-size. We are using the 'gzip-size' to do this.

## Changes
- updated verify-build-size.js
- add file-size-config.json
- removed the initial config from package.json

## Todos
-  Get more of a list of build file sizes that we're interested in and their threshold.

## How to test-drive this PR
- npm run prod:build
- change file-size-config.json to add/edit a file you would like to check if it's over the threshold
- npm run test:build-size
